cla_backend/apps/call_centre/forms.py

```diff
@@ -201,7 +201,7 @@ class CallMeBackForm(BaseCallMeBackForm):
     priority_callback = forms.BooleanField(required=False)
 
     def _is_dt_too_soon(self, dt):
-        return dt <= timezone.now() + datetime.timedelta(minutes=30)
+        return dt <= timezone.now() - datetime.timedelta(minutes=30)
```

Comment:
Isn't this change the opposite of what we want as it will not allow a datetime to be set unless it has been over 30mins from the current datetime?

Reply:
No to the first question, of what this does: it doesn't change the direction of the check (it's still "must be after) but the time it has to be after is now in the past not the future.

Yes to the second question, that is exactly what it would do. At 10:31, 10:00-10:30 is not valid but 10:30-11:00 is valid

Reply:
Ah okay, I think I understand now. Just to confirm, the first callback slot available at 10.31 would be 10.30 - 11am? On Slack, you said 11 - 11.30am

Reply:
I don't think I did.